packages/admin/Dockerfile

```diff
@@ -53,11 +53,19 @@ RUN apt-get update && apt-get install -y \
     libnss3 \
     libxss1 \
     fonts-liberation \
+    libgtk2.0-0 \
```

Comment:
I would like to separate these to another RUN statement, so we can keep cypress stuff separated.
Also, want to confirm if these requirements for running cypress locally, or are they only needed when we eventually run this remotely.